src/file/file.service.spec.ts

```diff
@@ -357,14 +357,37 @@ describe('File Logic', () => {
       expect(file2.parent).to.equal(file1.parent);
     });
 
-    it('should throw an error when KEY already exist', async () => {
+    it('should throw an error when KEY already exists', async () => {
       await FileService.create(bucket, 'tmp1', USER.id, 'text', null, KEY);
       await FileService.create(bucket, 'tmp2', USER.id, 'text', null, KEY)
       .should.eventually.be.rejectedWith(KeyAlreadyExistsError);
     });
 
   });
 
+  describe('#updateById', () => {
```

Comment:
How does updateById tests related to this issue/hotfix?

Reply:
I created tests that were missing and are related to this issue